ui/js/graphs.js

```diff
@@ -801,6 +814,71 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
         });
     };
 
+    var resolvePromise = $q.defer();
+    var addRelatedPlatforms = function(relatedSeries) {
+        PhSeries.getAllSeries(relatedSeries.projectName,
+            $scope.timeRange, optionCollectionMap).then(
+            function(seriesData) {
+                var platformList = seriesData.platformList;
```

Comment:
Makes sense-- maybe we could add an additional method in the future to just return the relevant signatures we want on the server side (by filtering on test name/suite, for example). We can do this later though.